redi-connect-front/src/components/organisms/RediHero.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {
+  Container,
+  Section,
+  Hero,
+  Columns,
+  Heading,
+  Content,
+} from "react-bulma-components";
+import Button from "../atoms/Button";
+import team from "../../assets/images/hero.svg";
+import teamMobile from "../../assets/images/hero-mobile.svg";
+import "./RediHero.scss";
+
+const RediHero = () => (
+  <Hero>
+    <Container>
+      <Section>
+        <Columns>
+          <Columns.Column size={5} className="hero__column">
+            <Heading size={1} renderAs="h1" className="hero__column--heading">
+              Welcome to ReDI Connect{" "}
+            </Heading>
+            <Columns.Column className="is-hidden-tablet">
+              <img src={teamMobile} alt="team" className="hero__column--img" />
+            </Columns.Column>
+            <Content size={4} className="hero__column--content">
+              Are you ready for the future of work? We connect thriving
+              professionals from the digital industry with students and alumni
+              of our Digital Career Program.{" "}
+            </Content>
+            <Button size="large" text="sign-up now!" />
+          </Columns.Column>
+          <Columns.Column offset={1} className="is-hidden-mobile">
+            <img src={team} alt="team" className="hero__column--img" />
```

Comment:
`hero__column--img` would mean that you have a `hero__column` block which has special styles with the `--img`
The correct way to apply the class is `hero__column__img`. This means you have `hero__column` which contains an image.